src/index.html

```diff
@@ -18,5 +23,87 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+
+    <header class="header">
+      <a
+        class="logo"
+        href="https://commons.wikimedia.org/wiki/File:LOGO_moyo_1.png"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="list">
+            <a
+              class="link link-blue is-active"
+              href="https://apple.com"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://samsung.com"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link"
+              href="https://sv.wikipedia.org/wiki/Smartmobil"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="list">
+            <a
+              class="link link-blue"
+              href="https://en.wikipedia.org/wiki/Laptop"
+              data-qa="hover"
+            >
+              LaptopsS&Computers
```

Comment:
The text within the `<a>` tag 'LaptopsS&Computers' has a typo and possible encoding issue with the ampersand. Consider changing it to 'Laptops & Computers' for clarity and proper HTML encoding.